pkg/factory/risingwave_object_factory.go

```diff
@@ -392,6 +392,7 @@ func (f *RisingWaveObjectFactory) argsForCompute() []string {
 		"compute-node",
 		"--config-path", "/risingwave/config/risingwave.toml",
 		"--host", fmt.Sprintf("$(POD_IP):%d", computePorts.ServicePort),
+		"--client-address", fmt.Sprintf("$(POD_NAME).%s.%s:%d", f.componentName(consts.ComponentCompute, ""), f.risingwave.Namespace, computePorts.ServicePort),
```

Comment:
I think we can leave the namespace out.

```suggestion
		"--client-address", fmt.Sprintf("$(POD_NAME).%s:%d", f.componentName(consts.ComponentCompute, ""), computePorts.ServicePort),
```

Reply:
Let me test in my cluster

Reply:
I have removed it